refml/topic_modeling/refml_sql.py

```diff
@@ -1,12 +1,61 @@
 from refml.topic_modeling.preprocess import stem_and_tokenize
 from refml.topic_modeling.model import refml_nmf_tsne
 import pandas as pd
+from sqlalchemy import create_engine
+from sqlalchemy.types import Integer, String, Float
+import cx_Oracle
 
-def runrefml(input, n_topics, n_top_words, n_dimensions, perplexity):
-    df['Text_Processed'] = input.apply(stem_and_tokenize)
-    return refml_nmf_tsne(df.Text_Processed, n_topics, n_top_words, n_dimensions, perplexity)
+def hero_to_df(Userid, Password, Host, Port, SID, query):
```

Comment:
let's rename this method `get_hero_conn` and it takes everything except the query and returns the `conn` object.  Also, you'll want to make sure to `conn.close()` somewhere, is this being done?